lib/TPP/CombineTpp.cpp

```diff
@@ -0,0 +1,128 @@
+//===- CombineTpp.cpp --------------------------------------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "TPP/Dialect/Tpp/TppOps.h"
+#include "TPP/Passes.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/MemRef/IR/MemRef.h"
+#include "mlir/IR/Matchers.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+using namespace mlir;
+
+#define GEN_PASS_CLASSES
+#include "TPP/Passes.h.inc"
+
+namespace {
+
+struct CombineTppOpPattern : public OpRewritePattern<tpp::ReluOp> {
+  using OpRewritePattern<tpp::ReluOp>::OpRewritePattern;
+
+  // TODO this is not a sufficient check.
+  static bool isVNNILayout(Operation *brgemmOp) {
```

Comment:
Don't we have this elsewhere? Should this check be in `TppUtils`?

Reply:
The other method was removed.